Dockerfile.deploy

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
+LABEL maintainer "Devtools <devtools@redhat.com>"
+LABEL author "Devtools <devtools@redhat.com>"
+ENV LANG=en_US.utf8
+ENV F8_INSTALL_PREFIX=/usr/local/devconsole-operator
+
+# Create a non-root user and a group with the same name: "devconsole-operator"
+ENV F8_USER_NAME=devconsole-operator
```

Comment:
maybe let's get rid out of `F8` prefix

Reply:
Fixed befe3ef.

Reply:
@michaelkleinhenz no it wasn't fixed in befe3ef. The code is still in there. We have `F8_` in various other locations and files as well if you take a look at the PR now. I suggest to keep the `F8_` and not replace it now. The reason is that environment variables marked with `F8_` are those that you should care about setting from the outside when running a Makefile for example: `F8_POSTGRES_HOST=foo make test-integration`. Can you revert the parts of befe3ef that modify the `F8_` bits please and keep the rest until we have a proper replacement for `F8_`?